### PR TITLE
CASMPET-5619 Add platform ca troubleshooting guide

### PR DESCRIPTION
**Pull request by kburns-hpe**
This adds documentation for the 3 common SSL failures due to applications not trusting the Platform CA.

### CODE REVIEW

File: troubleshooting/known_issues/platform_ca_issues.md [hunk 1]
```diff
@@ -0,0 +1,171 @@
+# Common Platform CA Issues
+
+## NCN Platform CA Certificate Does Not Match Certificate in BSS
+
+During install, if the beginning steps are re-run after the NCNs are booted
```
Comment (nrockershousen) [inline troubleshooting/known_issues/platform_ca_issues.md:5]:
```suggestion
During install, if the beginning steps are re-run after the NCNs are booted,
```

File: troubleshooting/known_issues/platform_ca_issues.md [hunk 2]
```diff
@@ -0,0 +1,171 @@
...
+then their platform-ca file will no longer match the server's CA. This can be
+detected with a goss test.
```
Comment (nrockershousen) [inline troubleshooting/known_issues/platform_ca_issues.md:7]:
```suggestion
detected with a Goss test.
```

File: troubleshooting/known_issues/platform_ca_issues.md [hunk 3]
```diff
@@ -0,0 +1,171 @@
...
+to equal
+    <int>: 0
+
+Total Duration: 0.058s
+Count: 1, Failed: 1, Skipped: 0
+```
+
+### Solution
+
+Run the following commands on any failed NCNs to update the platform-ca
```
Comment (nrockershousen) [inline troubleshooting/known_issues/platform_ca_issues.md:53]:
```suggestion
Run the following commands on any failed NCNs to update the `platform-ca` file.
```

File: troubleshooting/known_issues/platform_ca_issues.md [hunk 4]
```diff
@@ -0,0 +1,171 @@
...
+Command:
+
+```bash
+pip show ceritfy
+```
+
+Example Output:
+
+```text
+ncn-m001:~ # pip show certifi
```
Comment (nrockershousen) [inline troubleshooting/known_issues/platform_ca_issues.md:93]:
```suggestion
ncn-m001# pip show certifi
```

File: troubleshooting/known_issues/platform_ca_issues.md [hunk 5]
```diff
@@ -0,0 +1,171 @@
...
+Version: 2021.10.8
+Summary: Python package for providing Mozilla's CA Bundle.
+Home-page: https://certifiio.readthedocs.io/en/latest/
+Author: Kenneth Reitz
+Author-email: me@kennethreitz.com
+License: MPL-2.0
+Location: /root/.local/lib/python3.6/site-packages/certifi-2021.10.8-py3.6.egg
+Requires:
+Required-by: canu, kubernetes, requests
+ncn-m001:~ #
```
Comment (nrockershousen) [inline troubleshooting/known_issues/platform_ca_issues.md:104]:
```suggestion
ncn-m001#
```

File: troubleshooting/known_issues/platform_ca_issues.md [hunk 6]
```diff
@@ -0,0 +1,171 @@
...
+```
+
+### Solutions
+
+If certifi is installed in `/root/.local/...` then you can uninstall it by
+running
+
+```bash
+  pip uninstall certifi
```
Comment (nrockershousen) [inline troubleshooting/known_issues/platform_ca_issues.md:113]:
```suggestion
ncn-m001# pip uninstall certifi
```

File: troubleshooting/known_issues/platform_ca_issues.md [hunk 7]
```diff
@@ -0,0 +1,171 @@
...
+
+```bash
+systemctl restart cfs-state-reporter
+```
+
+## Certifi Has Been Updated and No Longer Respects the Local ca-bundle
+
+SLES ships a modified version of the python3 certifi module. This module
+uses the local ca-bundle.pem file. If certifi is updated, usually due to a pip
+install, then the ca bundle that certifi uses will revert to the one that's
```
Comment (nrockershousen) [inline troubleshooting/known_issues/platform_ca_issues.md:71]:
```suggestion
install, then the CA bundle that certifi uses will revert to the one that's
```

File: troubleshooting/known_issues/platform_ca_issues.md [hunk 8]
```diff
@@ -0,0 +1,171 @@
...
+
+curl failed to verify the legitimacy of the server and therefore could not
+establish a secure connection to it. To learn more about this situation and
+how to fix it, please visit the web page mentioned above.
+```
+
+### Solution
+
+Run the following commands to regenerate the ca-bundle.pem file with the
+platform-ca-certs.crt file included.
```
Comment (nrockershousen) [inline troubleshooting/known_issues/platform_ca_issues.md:159]:
```suggestion
`platform-ca-certs.crt` file included.
```

**Review by nrockershousen — CHANGES_REQUESTED** [review]

**Comment by kburns-hpe** [comment]
for the markdown-style-check, can we change MD024 allow_different_nesting to true so that it doesn't complain when too many headers match? I'd rather not add extra words to the headers just to avoid this check.

**Review by mharding-hpe — APPROVED** [review]

**Comment by mharding-hpe** [comment]
Needs a PR for main as well.

**Review by nrockershousen — APPROVED** [review]

**Pull request merged**